src/widgets/infinite-hits/infinite-hits.tsx

```diff
@@ -21,6 +21,7 @@ import {
   Hit,
   InsightsClientWrapper,
   Renderer,
+  InfiniteHitsCache,
```

Comment:
Would it make sense to define the `InfiniteHitsCache` type in the connector instead?

Reply:
Ah yeah, now I see the pattern.

Reply:
https://github.com/algolia/instantsearch.js/pull/4413/commits/312ddd3b69a2a7849ad5db4316484e7dc2693520